ONSdigital/census-rm-ops
# Use new action scheduler flow

# Motivation and Context
The flow for running a census survey has drastically changed, with many services removed or rewritten. The new process is much simpler, the prerequisite to load a sample into the case service is having an action plan set up. The home page is now the action plans page since this is the starting point for running the survey. This shows a list of existing action plans and provides a form to create more. The sample load and action rules pages are now accessed from an action plan page.

# What has changed
- Add views and logic for new action scheduler endpoints
- Redirect to action plans page from index
- Delete all now irrelevant pages (surveys, collection exercises etc)
- Use the new case queue and format in sample loader code
- Remove Redis caching from sample loader
- Use application config in Rabbit Context class
- Update unit tests
- Add unit test config class to ensure the unit tests cannot call actual services

# How to test?
Run against docker dev, you should be able to setup an action plan, load a sample and create an action rule and get the print file output.

# Links
https://trello.com/c/qnvevBVm/711-make-the-ops-tool-tests-work
